Mathlib/FieldTheory/SeparableDegree.lean

```diff
@@ -1,147 +1,297 @@
 /-
-Copyright (c) 2021 Jakob Scholbach. All rights reserved.
+Copyright (c) 2023 Jz Pan. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Jakob Scholbach
+Authors: Jz Pan
 -/
-import Mathlib.Algebra.Algebra.Basic
-import Mathlib.Algebra.CharP.ExpChar
+import Mathlib.FieldTheory.SplittingField.Construction
+import Mathlib.FieldTheory.IsAlgClosed.AlgebraicClosure
 import Mathlib.FieldTheory.Separable
-
-#align_import field_theory.separable_degree from "leanprover-community/mathlib"@"d11893b411025250c8e61ff2f12ccbd7ee35ab15"
+import Mathlib.FieldTheory.NormalClosure
+import Mathlib.RingTheory.IntegralDomain
+import Mathlib.RingTheory.Polynomial.SeparableDegree
 
 /-!
 
 # Separable degree
 
-This file contains basics about the separable degree of a polynomial.
+This file contains basics about the separable degree of a field extension.
+
+## Main definitions
+
+- `Emb F E`: the type of `F`-algebra homomorphisms from `E` to the algebraic closure of `E`.
```

Comment:
```suggestion
- `Field.Emb F E`: the type of `F`-algebra homomorphisms from `E` to the algebraic closure of `E`.
```
Are other names in this list also namespaced? If so, please add the namespace, so that documentation will linkify them.

Reply:
Namespaces added. But I assume that it's not necessary? In the old `SeparableDegree` file the `IsSeparableContraction` etc. is inside the `Polynomial` namespace. It's not prefixed in the docstring, but the link in the doc still works.